SquashMergeRequirements.md

```diff
@@ -7,10 +7,24 @@ When commiting your PR, please make sure to do these steps at Squash/Merge to as
 1. Tag the PR with the type of update: **Bug**, **Enhancement**, or **New-Feature**
 1. Tag the PR with **Azure API for FHIR** if this will release to the managed service
 1. Include a user friendly, 1-2 sentence in the Squash/Merge **description** wrapped at 72 characters
-1. Note if it **addresses a GitHub issue and/or a VSTS item** in the Squash/Merge description with a link (i.e. #1234 or AB#12345)
+1. Note if it **addresses a GitHub issue and/or a VSTS item** in the Squash/Merge description (i.e. #1234 or AB#12345)
 
 Sometimes our changes will create breaking changes or may need a warning associated with them. If this is the case, take these additional steps:
 
 1. **Create a new issue** in GitHub that describes the warning/issue
 1. Tag the PR with **KI-Breaking** or **KI-Warning**
 1. **Reference the new GitHub issue** from the PR description
+
+## Example
+
+Squash/Merge commit heading:
+
+PR title + PR number (maximum 50 characters)
+
+`Adds CMK option to deployment templates. (#1234)`
+
+Squash/Merge description:
+
+User friendly description + GitHub or VSTS item number (maximum 72 characters)
+
+`Enables customer managed keys for Cosmos DB backed services. (AB#12345)`
```

Comment:
How does the semver command fit into the 72 character limit? 
I think having one of those commands in the example would be helpful.

Reply:
72 chars is the recommending line-wrap limit, you can have multiple lines, for example if you wanted to do:

```
Enables customer managed keys for Cosmos DB backed services. 
Refs AB#12345
+semver: patch
```

That would be ok

Reply:
Ah, that makes sense. Thanks! I will add this to the doc.

Reply:
Updated in 0074627.